static/js/compose.js

```diff
@@ -495,53 +494,43 @@ export function initialize() {
         },
     );
 
-    $("#compose_invite_users").on("click", ".compose_invite_link", (event) => {
-        event.preventDefault();
-
-        const $invite_row = $(event.target).parents(".compose_invite_user");
-
-        const user_id = Number.parseInt($invite_row.data("user-id"), 10);
-        const stream_id = Number.parseInt($invite_row.data("stream-id"), 10);
-
-        function success() {
-            const $all_invites = $("#compose_invite_users");
-            $invite_row.remove();
-
-            if ($all_invites.children().length === 0) {
-                $all_invites.hide();
-            }
-        }
+    $("#compose_banners").on(
+        "click",
+        `.${compose_error.CLASSNAMES.recipient_not_subscribed} .compose_banner_action_button`,
+        (event) => {
+            event.preventDefault();
 
-        function failure(error_msg) {
-            clear_invites();
-            compose_error.show_error_message(
-                error_msg,
-                compose_error.CLASSNAMES.generic_compose_error,
-                $("#compose-textarea"),
+            const $invite_row = $(event.target).parents(
+                `.${compose_error.CLASSNAMES.recipient_not_subscribed}`,
```

Comment:
This could be just `.parents("compose_banner")`; might be worth doing just to avoid unnecessary references to the CLASSNAMES lookup.

Reply:
sounds good, and also changed in other relevant place